mbed_tools_ci_scripts/tag_and_release.py

```diff
@@ -49,10 +53,11 @@ def tag_and_release(mode: CommitType, current_branch: Optional[str] = None) -> N
     if mode == CommitType.DEVELOPMENT:
         return
     _update_documentation()
+    licensing = _update_licensing_summary()
     add_licence_header(0)
     _update_repository(mode, is_new_version, version, current_branch)
     if is_new_version:
-        _generate_spdx_reports()
+        _generate_spdx_reports(licensing)
```

Comment:
Is `licensing` a `Path`? Might be worth improving the naming if so.

Reply:
Oh ignore me, I was looking at the definition in the file above.

Reply:
Ok, I'm still confused. `_generate_spdx_reports` takes `project` as argument. What is `licensing`? 😅

Reply:
I renamed the parameter to be clearer

Reply:
aee6f7ee5b8763b2bacc9e5ff6f6923158220e52